### DISCUSSION TITLE
AlphaTims on macOS and M1

### DISCUSSION THREAD

**Discussion by DarylWM** [MannLabs/alphatims#211] [Ideas]
Hello. I'm keen to use AlphaTims on macOS. Have you had any success with this? I know the calibration algorithm relies on Bruker shared libraries, and only Windows and Linux libraries have been made available. We can use the mobility and m/z estimates, but it would be more consistent with other platforms to use the Bruker libraries. Can we convince Bruker to build the shared library for macOS/M1?

**Comment by swillems** [comment]
Hey Daryl. My main laptop actually is a macbook and all development is actually done there. Windows and Linux are just checked afterwards for compatability. I recently implemented 2 small bugfixes relating to OS differences, but generally I have little to no problems on any platform.

If you really want consistency, the simple solution would be to export an hdf on windows/linux and then use this .hdf file on a macOS. This way there are no differences (Bruker SDK is only used once when defining reference arrays) between different OS's. As you already mention, making the SDK available is something only Bruker can do, so you would have to ask them directly.

**Comment by DarylWM** [comment]
Thanks Sander. Yes it's just the calibration algorithms in the Bruker library that would be nice to have on macOS. I haven't noticed any other differences across platforms.

I'll ask if Bruker would be interested in releaseing a macOS build of the shared library. In the meantime that's a good idea to copy the HDF over from Linux; I didn't think of that.
